src/ansys/fluent/core/services/batch_ops.py

```diff
@@ -2,28 +2,34 @@
 
 import inspect
 import logging
-from typing import List, Tuple
+from types import ModuleType
+from typing import Optional, TypeVar
 import weakref
 
+from google.protobuf.message import Message
 import grpc
 
 import ansys.api.fluent.v0 as api
 from ansys.api.fluent.v0 import batch_ops_pb2, batch_ops_pb2_grpc
 from ansys.fluent.core.services.error_handler import catch_grpc_error
 
-network_logger = logging.getLogger("pyfluent.networking")
+_TBatchOps = TypeVar("_TBatchOps", bound="BatchOps")
+
+network_logger: logging.Logger = logging.getLogger("pyfluent.networking")
 
 
 class BatchOpsService:
     """Class wrapping methods in batch RPC service."""
 
-    def __init__(self, channel: grpc.Channel, metadata: List[Tuple[str, str]]):
+    def __init__(self, channel: grpc.Channel, metadata: list[tuple[str, str]]) -> None:
```

Comment:
Don't know if this already has been mentioned somewhere else and I missed it, but these changes from `typing.List` to `list` shouldn't cause any issues, as we dropped support for Python 3.8, and these are a standard part of Python 3.9, right?

In this case we could clean up a lot of deprecated `typing` imports that we had been using to support Python 3.8 (there are many instances)